.github/workflows/multi-arch_compliance_operator_build.yml

```diff
@@ -0,0 +1,80 @@
+name: Multiarch build
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v**
+
+env:
+  IMAGE_NAME: compliance-operator
+  IMAGE_TAG: multi-arch
+  IMAGE_REGISTRY: ghcr.io
+  IMAGE_NAMESPACE: complianceascode
+
+jobs:
+  build:
+    name: Build and push multi-arch image
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: add checkout action...
+        uses: actions/checkout@v2
+
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Build image linux/amd64
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.IMAGE_NAME }}
+          tags: linux-amd64
+          arch: amd64
+          containerfiles: |
+            ./build/Dockerfile
+
+      - name: Build image linux/ppc64le
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.IMAGE_NAME }}
+          tags: linux-ppc64le
+          arch: ppc64le
+          containerfiles: |
+            ./build/Dockerfile
+
+      - name: Build image linux/s390x
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.IMAGE_NAME }}
+          tags: linux-s390x
```

Comment:
Are consumers planning to use these tags and expect updates automatically? Typically, when we tag a version we use the version as part of the tag name. Wouldn't this result in a single image being overwritten with the same tag name each time this runs?

https://github.com/ComplianceAsCode/compliance-operator/blob/master/.github/workflows/release-latest.yml#L15
https://github.com/ComplianceAsCode/compliance-operator/blob/master/.github/workflows/release.yml#L14

Reply:
I have added required changes and tested successfully as well. 
please let me know for any more changes.

Reply:
Is this supposed to be ${{ env.RELEASE_TAG }}-linux-s390x like the other two?

When this runs on master, what does env.RELEASE_TAG evaluate to, latest?

Reply:
Hi @rhmdnd ,

As per your suggestion we have updated code with changes.
Approach folllowed:
**1. When code is pushed to master branch, it will build image with latest tag.
2. For newer release tag, image will be build with release tag.**
For this we have created two seperate workflows as per the example you shared above.

So we will have two different files for each image. i.e. for compliance-operator and openscap image.

Compliance operator workflow files:
multi-arch_compliance_operator_release-latest.yml
multi-arch_compliance_operator_release.yml

Openscap workflow files:
multi-arch_openscap_release-latest.yml
multi-arch_openscap_release.yml

Please let me know for any more changes.